ERROR - 2023-08-08 23:59:46 --> Error connecting to the database: mysqli_sql_exception: Unknown database 'pp2023_ciudadanos' in C:\xampp\htdocs\actividad1\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\xampp\htdocs\actividad1\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'pp2023_ciudadan...', 3306, '', 0)
#1 C:\xampp\htdocs\actividad1\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\actividad1\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\actividad1\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\actividad1\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\actividad1\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\actividad1\app\Controllers\RegionController.php(13): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\actividad1\system\CodeIgniter.php(932): App\Controllers\RegionController->index()
#8 C:\xampp\htdocs\actividad1\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegionController))
#9 C:\xampp\htdocs\actividad1\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\actividad1\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Unknown database 'pp2023_ciudadanos' in C:\xampp\htdocs\actividad1\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\actividad1\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\actividad1\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\actividad1\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\actividad1\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\actividad1\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\actividad1\app\Controllers\RegionController.php(13): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\actividad1\system\CodeIgniter.php(932): App\Controllers\RegionController->index()
#7 C:\xampp\htdocs\actividad1\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegionController))
#8 C:\xampp\htdocs\actividad1\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\actividad1\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-08-08 23:59:46 --> Unable to connect to the database.
Main connection [MySQLi]: Unknown database 'pp2023_ciudadanos'
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `regiones`', [], false)
 3 SYSTEMPATH\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\RegionController.php(13): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\RegionController->index()
 7 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegionController))
 8 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
